gyucheol/4일차/로또의_최고_순위와_최저_순위.js

```diff
@@ -0,0 +1,11 @@
+function solution(lottos, win_nums) {
+    let answer = [];
+    const win = lottos.filter(lottos => win_nums.includes(lottos)).length;
+    const zero = lottos.filter(lottos => lottos === 0).length;
+
+    const min = 7 - win >= 6 ? 6 : 7 - win;
+    const max = min - zero < 1 ? 1 : min - zero;
+
+    answer = [max, min];
+    return answer;
```

Comment:
```suggestion
    let answer = [];
    ...
    
    ...
    answer = [max, min];
    return answer;
    
    ->
   const answer = [];
   
   answer.push(max, min);
   return answer;
   
   or
   
   return [max, min];
```
let answer = []로 배열을 선언하면 push 메서드를 이용해서 값을 넣어줘야 합니다. 즉 answer = [max, min]는 재할당이므로 메모리 주소값이 바뀝니다. 데이터의 무결성을 지키기 위해 배열이나 객체는 보통 상수(const)로 선언하고 요소의 값을 바꿉니다.